extensions/git/package.json

```diff
@@ -2543,6 +2543,15 @@
           "default": false,
           "markdownDescription": "%config.mergeEditor%",
           "scope": "window"
+        },
+        "git.experimental.optimisticUpdate": {
```

Comment:
Given we have tags to indicate the experimental status, should this just be called `git.optimisticUpdate`?

Reply:
Fixed.